tests/contrib/mysqldb/test_mysqldb.py

```diff
@@ -25,7 +26,7 @@ def tearDown(self):
         if self.conn:
             try:
                 self.conn.ping()
-            except MySQLdb.InterfaceError:
+            except (MySQLdb.InterfaceError, MySQLdb.OperationalError):
```

Comment:
why this change here?

Reply:
hmm not sure. Maybe @majorgreys was getting this locally? Seems like a pretty harmless change. I don't even know what this code is doing or why it's needed in the first place.

Reply:
> Seems like a pretty harmless change

Yeah, just curious if it needs to be in this PR or on it's own